packages/exoflex/src/components/Switch.tsx

```diff
@@ -22,6 +24,21 @@ export type SwitchProps = AccessibilityProps & {
 
 const MARGIN = 2;
 
+export function getTrueWidth(
+  baseWidth: number,
+  style?: StyleProp<ViewStyle>,
+): number {
+  let flattenedStyle = StyleSheet.flatten(style);
+  let width = +(flattenedStyle?.width ?? baseWidth);
```

Comment:
Maybe we should call `toNumber` here too?
```suggestion
  let width = toNumber(flattenedStyle?.width ?? baseWidth);
```

Reply:
It's the same thing actually, `toNumber` also uses `+` when returning the value. But for better readability, I guess we should change it.